thirdeye/thirdeye-frontend/tests/helpers/resolver.js

```diff
@@ -0,0 +1,11 @@
+import Resolver from '../../resolver';
+import config from '../../config/environment';
```

Comment:
It's usually better to use absolute path to be sure that we import from the right place. Also, it's more readable (or clear) where we're importing from.